packages/twilio-run/src/config/env/env-import.ts

```diff
@@ -79,18 +79,6 @@ export async function getConfigFromFlags(
 
   let serviceName = await getServiceNameFromFlags(flags);
 
-  if (!flags.key) {
-    throw new Error(
-      'Missing --key argument. Please provide a key for your environment variable.'
-    );
-  }
-
-  if (!flags.value) {
-    throw new Error(
-      'Missing --value argument. Please provide a key for your environment variable.'
```

Comment:
Should this be value?

Reply:
They are both not used so I'm just deleting the checks. They were copy & paste from the env-set.ts file which actually needs them.

Reply:
Oh nvm, I thought you were adding this 🙈